src/Microsoft.Health.Dicom.Api.UnitTests/Controllers/ExtendedQueryTagControllerTests.cs

```diff
@@ -64,11 +64,50 @@ public async Task GivenFeatureIsDisabled_WhenCallingApi_ThenShouldThrowException
                 NullLogger<ExtendedQueryTagController>.Instance);
 
             await Assert.ThrowsAsync<ExtendedQueryTagFeatureDisabledException>(() => controller.GetTagAsync(DicomTag.PageNumberVector.GetPath()));
+            await Assert.ThrowsAsync<ExtendedQueryTagFeatureDisabledException>(() => controller.GetTagErrorAsync(DicomTag.PageNumberVector.GetPath()));
             await Assert.ThrowsAsync<ExtendedQueryTagFeatureDisabledException>(() => controller.GetAllTagsAsync());
             await Assert.ThrowsAsync<ExtendedQueryTagFeatureDisabledException>(() => controller.PostAsync(Array.Empty<AddExtendedQueryTagEntry>()));
             await Assert.ThrowsAsync<ExtendedQueryTagFeatureDisabledException>(() => controller.DeleteAsync(DicomTag.PageNumberVector.GetPath()));
         }
 
+        [Fact]
+        public async Task GivenTagPath_WhenCallingApi_ThenShouldReturnOk()
+        {
+            IMediator mediator = Substitute.For<IMediator>();
+            string path = "11330001";
+
+            var controller = new ExtendedQueryTagController(
+                mediator,
+                Options.Create(new FeatureConfiguration { EnableExtendedQueryTags = true }),
+                NullLogger<ExtendedQueryTagController>.Instance);
+            controller.ControllerContext.HttpContext = new DefaultHttpContext();
+
+            var expected = new GetExtendedQueryTagErrorsResponse(
+                new List<ExtendedQueryTagError>
+                {
+                    new ExtendedQueryTagError(DateTime.UtcNow, Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), "Error 1"),
+                    new ExtendedQueryTagError(DateTime.UtcNow, Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), "Error 2"),
+                    new ExtendedQueryTagError(DateTime.UtcNow, Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), "Error 3"),
+                });
+
+            mediator
+                .Send(
+                    Arg.Is<GetExtendedQueryTagErrorsRequest>(x => x.Path == path),
```

Comment:
nit: this alignment is off. The 2nd argument and the `Returns` method are idented one extra level